helm/helm-2to3-migration-app/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: 1.0.0
```

Comment:
Can we use the version of helm-2to3 here?

Reply:
Sure, at the moment I used it as 1.0.4. Will change it.

Reply:
Oh, I think you mean the version of 2to3 plugin, then it's 0.5.0